subsys/bluetooth/audio/csip_internal.h

```diff
@@ -21,5 +21,30 @@ struct bt_csip_set_sirk {
 	uint8_t value[BT_CSIP_SET_SIRK_SIZE];
 } __packed;
 
+struct bt_csip_set_coordinator_svc_inst *lookup_instance_by_index(const struct bt_conn *conn,
+								  uint8_t idx);
```

Comment:
I guess it's OK to move this to a header file so it can be used for the BT tester, but in that case we should add proper prefix to it:
```suggestion
struct bt_csip_set_coordinator_svc_inst *bt_csip_set_coordinator_lookup_instance_by_index(const struct bt_conn *conn,
								  uint8_t idx);
```

Reply:
Done